aspnetcore/blazor/tooling.md

```diff
@@ -104,68 +116,53 @@ Select **Next**.
-
-:::moniker range="< aspnetcore-8.0"
-
-> [!IMPORTANT]
-> When executing a hosted Blazor WebAssembly app, run the app from the solution's **:::no-loc text="Server":::** project.
->
-> When the app is launched, only the `Properties/launchSettings.json` file in the :::no-loc text="Server"::: project is used.
-
-:::moniker-end
-
 :::zone-end
 
-:::zone pivot="linux-macos"
+:::zone pivot="vsc"
 
-To create a Blazor app on Linux or macOS, use the following guidance:
+[Visual Studio Code](https://code.visualstudio.com) is an open source, cross-platform Integrated Development Environment (IDE) that can be used to develop Blazor apps.
 
-Use the [.NET command-line interface (CLI)](/dotnet/core/tools/) to execute commands in a command shell.
+Install the latest version of [Visual Studio Code](https://code.visualstudio.com/Download) for your platform.
 
-Install the latest version of the [.NET Core SDK](https://dotnet.microsoft.com/download). If you previously installed the SDK, you can determine your installed version by executing the following command:
+The Visual Studio Code (VS Code) instructions for ASP.NET Core development in this tutorial use the [.NET CLI](/dotnet/core/tools/), which is part of the .NET SDK. .NET CLI commands are issued in VS Code's integrated [**Terminal**](https://code.visualstudio.com/docs/editor/integrated-terminal), which is a [PowerShell command shell](/powershell/). The **Terminal** is opened by selecting **New Terminal** from the **Terminal** menu in the menu bar.
 
-```dotnetcli
+Install the [.NET SDK](https://dotnet.microsoft.com/download). If you previously installed one or more .NET SDKs and want to see your active version, execute the following command in the **Terminal**:
+
+```powershell
 dotnet --version
 ```
 
-Install the latest version of [Visual Studio Code](https://code.visualstudio.com) for your platform.
-
 Install the [C# Dev Kit for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit). For more information, see <xref:blazor/debug#visual-studio-code-prerequisites>.
 
+If you're new to VS Code, see the [VS Code documentation](https://code.visualstudio.com/docs). If you're new to the .NET SDK, see [What is the .NET SDK?](/dotnet/core/sdk) and the associated articles in the .NET SDK documentation.
+
 :::moniker range=">= aspnetcore-8.0"
 
 Create a new project:
 
-* For a Blazor Web App experience with default interactive server-side rendering, execute the following command in a command shell that uses the `blazor` project template:
+* Change to the directory using the `cd` command to where you want to create the project folder (for example, `cd c:/users/Bernie_Kopell/Documents`).
 
-  ```dotnetcli
+* For a Blazor Web App experience with default interactive server-side rendering (interactive SSR), execute the following command in a command shell that uses the `blazor` project template:
```

Comment:
For Visual Studio Code, there is a command pallet experience for creating project templates that we should probably use: https://code.visualstudio.com/docs/csharp/get-started#_create-a-hello-world-app

Reply:
This is another change that I'll apply to the Blazor tutorial PR.

Reply:
I don't understand part of their guidance ...

> go to the Explorer view and select Create .NET Project.

I don't see that.

I'll base our text on using the Command Palette.

Reply:
***Done!*** 👍 ... Will be on the next commit.